WriteMissingResourceItems/Program.cs

```diff
@@ -43,31 +43,39 @@ private static Dictionary<CultureInfo, Dictionary<string, string>> ReadExcel(str
     {
         var result = new Dictionary<CultureInfo, Dictionary<string, string>>();
 
-        using (var workbook = new XLWorkbook(filePath))
+        try
         {
-            var worksheet = workbook.Worksheet(1);
-            var headerRow = worksheet.Row(1);
 
-            for (int col = 2; col <= headerRow.LastCellUsed().Address.ColumnNumber; col++)
+            using (var workbook = new XLWorkbook(filePath))
             {
-                var cultureName = headerRow.Cell(col).GetComment().Text;
-                var culture = CultureInfo.GetCultureInfo(cultureName);
+                var worksheet = workbook.Worksheet(1);
+                var headerRow = worksheet.Row(1);
 
-                ref var value = ref CollectionsMarshal.GetValueRefOrAddDefault(result, culture, out bool exists);
-                if (!exists)
-                    value = [];
-
-                foreach (var row in worksheet.RowsUsed().Skip(1))
+                for (int col = 2; col <= headerRow.LastCellUsed().Address.ColumnNumber; col++)
                 {
-                    var key = row.Cell(1).GetValue<string>();
-                    var translation = row.Cell(col).GetValue<string>();
-                    if (string.IsNullOrWhiteSpace(translation) || translation == "-")
-                        continue;
-
-                    value![key] = translation;
+                    var cultureName = headerRow.Cell(col).GetComment().Text;
+                    var culture = CultureInfo.GetCultureInfo(cultureName);
+
+                    ref var value = ref CollectionsMarshal.GetValueRefOrAddDefault(result, culture, out bool exists);
+                    if (!exists)
+                        value = [];
+
+                    foreach (var row in worksheet.RowsUsed().Skip(1))
+                    {
+                        var key = row.Cell(1).GetValue<string>();
+                        var translation = row.Cell(col).GetValue<string>();
+                        if (string.IsNullOrWhiteSpace(translation) || translation == "-")
+                            continue;
+
+                        value![key] = translation;
+                    }
                 }
             }
         }
+        catch (Exception ex)
+        {
+            Console.WriteLine($"Could not open excel-file: {filePath}, reason: {ex.Message}");
```

Comment:
I think it is better to add this catch in the main function and catch errors from all parts of the code. Adding it here makes the program continue execution possibly with a half full dictionary which will be exported to the resx.